src/Tests/Mapping/Types/Core/Range/IpRange/IpRangePropertyTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using Nest;
+using Tests.Framework;
+using Tests.Framework.Integration;
+using Tests.Framework.ManagedElasticsearch.Clusters;
+using Tests.Framework.MockData;
+
+namespace Tests.Mapping.Types.Core.Range.IpRange
+{
+	[SkipVersion("<5.5.0", "ip range type is a new 5.5.0 feature")]
+	public class IpRangePropertyTests : PropertyTestsBase
```

Comment:
`IpAddressRangePropertyTests`  for consistency?

Reply:
:-1: